db/sqlc/migrations/000004_privacy_pairs.up.sql

```diff
@@ -0,0 +1,26 @@
+-- privacy_pairs stores the privacy map pairs for a given session group.
+CREATE TABLE IF NOT EXISTS privacy_pairs (
+    -- The group ID of the session that this privacy pair is associated
+    -- with.
+    group_id BIGINT NOT NULL REFERENCES sessions(id) ON DELETE CASCADE,
+
+    -- The key of the privacy pair.
+    real TEXT NOT NULL,
```

Comment:
nit:
real is seen as reserved keyword/datatype by many sql syntax highlighting tools (as also seen here in the Github uI), so would potentially choose another name as it gets a bit confusing.